doc/concepts/embeddings.rst

```diff
@@ -0,0 +1,45 @@
+.. _embeddings:
+
+Quantum Embeddings
+===================
+
+A *Quantum Embedding* is the process of representing classical data as quantum states in a Hilbert space, i.e., *a quantum feature map*. An embedding takes a classical datapoint :math:`x` and translates it into a set of gate parameters in a quantum circuit. This ends up creating a state :math:`\mid \psi_x \rangle` on the quantum system. This process is a crucial part of designing quantum algorithms and hence their complexity. For more details, see :cite:`Mariabook2018` and :cite:`schuld2018quantum`. PennyLane provides built-in embedding templates that can do the embedding for the user; see :ref: `<pennylane.templates.embeddings>`. 
```

Comment:
```suggestion
A *Quantum Embedding* represents classical data as quantum states in a Hilbert space, i.e., *a quantum feature map*. An embedding takes a classical datapoint :math:`x` and translates it into a set of gate parameters in a quantum circuit. This ends up creating a quantum state :math:`\mid \psi_x \rangle`. This process is a crucial part of designing quantum algorithms and affects their computational power. For more details, see :cite:`Mariabook2018` and :cite:`schuld2018quantum`. PennyLane provides built-in embedding templates that can do the embedding for the user; see :ref: `<pennylane.templates.embeddings>`. 
```

Reply:
Unresolving this because there is a second problem: the :ref: is not rendered correctly in browser.

Reply:
yep on it :)